components/eam/bld/namelist_files/use_cases/2010_scream_hr.xml

```diff
@@ -123,6 +127,11 @@
 <se_tstep hgrid="ne256np4"> 37.5 </se_tstep>
 <dt_tracer_factor hgrid="ne256np4"> 8 </dt_tracer_factor>
 <hypervis_subcycle_q hgrid="ne256np4"> 8 </hypervis_subcycle_q>
+<!-- Settings specific for ne1024 configuration -->
+<se_tstep hgrid="ne1024np4"> 8.333333333333333d0 </se_tstep>
+<dt_remap_factor hgrid="ne1024np4"> 6 </dt_remap_factor>
```

Comment:
I think dt_remap_factor should be 2, not 6

Reply:
taking the whole se_tstep out since, #1686 will do that in namelist_defaults_eam.xml